docs/messages_guide/src/asciidoc/_chapters/ddl_msgs.adoc

```diff
@@ -761,40 +814,42 @@ constraint specified in the CREATE INDEX statement.
 change the rows in the table to remove duplicates. Then reissue the
 statement.
 
-<<<
-[[SQL-1057]]
-== SQL 1057
+[[SQL-1055]]
+== SQL 1055
 
 ```
-{project-name} objects cannot be created on <volume-name>: File system error <error-number>.
+Object <privilege-object> already exists.
 ```
 
-Where <volume-name> is the name of a volume.
+Where <privilege-object> is an object you specified on a REGISTER statement.
 
-Where <error-number> is the file system error number.
+*Cause:* You attempted to register a new privilege object in the 
```

Comment:
Actually, a <privilege-object> is a component object name.  So perhaps:
"Where <privilege-object> is a component object name you specified for a REGISTER COMPONENT statement."
"Cause: You attempted to register a new component in the {project-name} database but this component object has already been registered."
...
"Recovery:  If the component object name is incorrect, correct and resubmit."